gandalf.js

```diff
@@ -1,24 +1,22 @@
-const isPalindrome = word => word === word.split("").reverse().join("");
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.